shelter/src/index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Shelter</title>
+    <link rel="stylesheet" href="normalize.css" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <p class="header__title">Cozy House</p>
+        <p class="header__subtitle">Shelter for pets in Boston</p>
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item header__item--active">
+            <a href="#" class="header__link">About the shelter</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Our pets</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Help the shelter</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="intro">
+        <h1>Shelter Cozy House</h1>
+        <div class="intro__info">
+          <h2 class="intro__title">Not only people need a house</h2>
+          <p class="intro__description">
+            We offer to give a chance to a little and nice puppy with an
+            extremely wide and open heart. He or she will love you more than
+            anybody else in the world, you will see!
+          </p>
+          <button class="button" type="button">Make a friend</button>
+        </div>
+        <div class="intro__image">
+          <img
+            class="intro__photo"
+            src="images/start-screen-puppy.png"
+            alt="puppy"
+            width="698"
+            height="728"
+          />
+        </div>
+      </section>
+      <section class="about">
+        <div class="about__image">
+          <img
+            src="images/about-pets.png"
+            alt="about-pets"
+            width="300"
+            height="408"
+          />
+        </div>
+        <div class="about_content">
+          <h3 class="title about_title">About the shelter “Cozy House”</h3>
```

Comment:
```suggestion
          <h3 class="about_title title">About the shelter “Cozy House”</h3>
```

here and other places